applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp

```diff
@@ -2154,13 +2154,22 @@ ShellCrossSection::SectionBehaviorType ShellThickElement3D3N::GetSectionBehavior
 void ShellThickElement3D3N::save(Serializer& rSerializer) const
 {
     KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, BaseShellElement);
-    rSerializer.save("CTr", mpCoordinateTransformation);
+    bool is_derived = (nullptr != dynamic_cast<ShellT3_CorotationalCoordinateTransformation*>(mpCoordinateTransformation.get()));
```

Comment:
Naming: `is_corotational` would describe better what is happening here. Especially in the load functions, where you decide if a corotational or a "base" transformation is created.